src/styles/unstyled.anchor.ts

```diff
@@ -0,0 +1,16 @@
+import { styled } from '@linaria/react'
+
+export const ElUnstyledAnchor = styled.a`
```

Comment:
Can this be removed now?

Reply:
I did wonder why you wanted it if it weren't to be used? Am sure you asked for a component we could extend from?